bokehjs/src/lib/models/glyphs/glyph.ts

```diff
@@ -368,7 +368,17 @@ export abstract class GlyphView extends DOMComponentView {
     }
   }
 
-  protected _transform_array<T>(prop: p.BaseCoordinateSpec<T>, array: Arrayable<unknown>) {
+  protected transform_array<T>(prop: p.BaseCoordinateSpec<T>, array: Arrayable<unknown>) {
```

Comment:
Why rename this? Protected and private members are prefixed with `_` by convention.

Reply:
I thought a benefit of having actual `private` and `protected` capability in the language was that typographic conventions became unnecessary. I wanted to keep `_some_name` out of the visible log messages.

Reply:
This is still JavaScript underneath, so the convention applies to that. There is the new `#` prefix for private members (though that comes with a lot of baggage), but nothing for protected members.

> I wanted to keep _some_name out of the visible log messages.

That sounds reasonable, but I also doubt that it's a big concern for an average user, who is very likely not to know about the JS console at all. With issue #14252 resolved, 99% of users will never see that, maybe except when asked explicitly when submitting a bug report.

Reply:
[1baf82f](https://github.com/bokeh/bokeh/pull/14308/commits/1baf82f1c05ce80a3b7f02b4494038d4f0394325)